src/components/Calculator.js

```diff
@@ -19,25 +18,24 @@ const [result, setResult] = useState({ total: solution, next: 0, operation: '' }
-                <button type="button" onClick={clickFunction} value="+" className="buttonClick button divisionBtn extraBtn">+</button>
-                <button type="button" onClick={clickFunction} value="7" className="buttonClick button">7</button>
-                <button type="button" onClick={clickFunction} value="8" className="buttonClick button">8</button>
-                <button type="button" onClick={clickFunction} value="9" className="buttonClick button">9</button>
-                <button type="button" onClick={clickFunction} value="x" className="buttonClick button muplyBtn extraBtn">x</button>
-                <button type="button" onClick={clickFunction} value="4" className="buttonClick button">4</button>
-                <button type="button" onClick={clickFunction} value="5" className="buttonClick button">5</button>
-                <button type="button" onClick={clickFunction} value="6" className="buttonClick button">6</button>
-                <button type="button" onClick={clickFunction} value="/" className="buttonClick button additionBtn extraBtn">/</button>
-                <button type="button" onClick={clickFunction} value="1" className="buttonClick button">1</button>
-                <button type="button" onClick={clickFunction} value="2" className="buttonClick button">2</button>
-                <button type="button" onClick={clickFunction} value="3" className="buttonClick button">3</button>
-                <button type="button" onClick={clickFunction} value="-" className="buttonClick button minusBtn extraBtn">-</button>
-                <button type="button"  onClick={clickFunction} value="0" className="buttonClick button zero">0</button>
-                <button type="button" onClick={clickFunction} value="." className="buttonClick button">.</button>
+                <button type="button" onClick={clickFunction} value="%" className="buttonClick  percentBtn functBtn">%</button>
+                <button type="button" onClick={clickFunction} value="+" className="buttonClick  divisionBtn extraBtn">+</button>
+                <button type="button" onClick={clickFunction} value="7" className="buttonClick ">7</button>
+                <button type="button" onClick={clickFunction} value="8" className="buttonClick ">8</button>
+                <button type="button" onClick={clickFunction} value="9" className="buttonClick ">9</button>
+                <button type="button" onClick={clickFunction} value="x" className="buttonClick  muplyBtn extraBtn">x</button>
+                <button type="button" onClick={clickFunction} value="4" className="buttonClick ">4</button>
+                <button type="button" onClick={clickFunction} value="5" className="buttonClick ">5</button>
+                <button type="button" onClick={clickFunction} value="6" className="buttonClick ">6</button>
+                <button type="button" onClick={clickFunction} value="/" className="buttonClick  additionBtn extraBtn">/</button>
```

Comment:
Hi @henrycode460 
Great job implementing the changes as per project requirements.

- [ ] Kindly check this button, your app generates an error every time I try to use this operation.
![divided](https://user-images.githubusercontent.com/55596970/178642021-7819c4b6-00a1-43b3-b461-4c35316e57e9.png)